www/src/pages/foundations/responsive.tsx

```diff
@@ -99,6 +124,57 @@ function Responsive({ pageContext }) {
         <CodeBlock className="language-scss">
```

Comment:
On this page I am not sure if it is worth deleting information that relates to SCSS variables. Any thoughts?